docs/HowTo/Configure/Consensus-Protocols/Add-Validators.md

```diff
@@ -468,7 +468,7 @@ To bypass the smart contract and specify new validators:
                "blockperiodseconds": 2,
                "epochlength": 30000,
                "requesttimeoutseconds": 4,
-               “validatorcontractaddress”: “0x0000000000000000000000000000000000007777”
+               "validatorcontractaddress”: “0x0000000000000000000000000000000000007777"
```

Comment:
```suggestion
               "validatorcontractaddress": "0x0000000000000000000000000000000000007777"
```

Reply:
fixed that one and a few more. Is there a way to find and replace these special characters?

Reply:
Yes IntelliJ distinguishes between the two characters. Find and replace using shift + cmd + R.